pages/common/javap.md

```diff
@@ -0,0 +1,25 @@
+# javap
+
+> Disassemble one or more class files and list them.
+> Part of `JDK`.
```

Comment:
Why this?

Reply:
Javap isn't an independent command. From what I remember it won't work with OpenJDK or it can't be installed independently. It is a part of the official JDK package.

I think we can rephrase this a bit. Any suggestions?

Reply:
Yeah, it's included in the OpenJDK

Reply:
Thanks, I will remove that line.